.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
+
+on: [push, pull_request]
+
```

Comment:
Would it be better to do pushes to `main`, PRs, and then the `workflow_dispatch` for manual kick-offs?
Might burn a bunch of unnecessary compute running on every push, at the cost of requiring someone to manually run them periodically while reviewing a PR.

Reply:
Regardless I think `workflow_dispatch` might be useful if we ever need to test changes by running from new feature branches.

Reply:
resolved